src/components/menu/component.test-pw.tsx

```diff
@@ -304,6 +304,89 @@ export const MenuComponentFullScreen = (
   );
 };
 
+export const MenuComponentFullScreenOpen = () => {
```

Comment:
**question**: could we potentially utilise the existing `MenuComponentFullScreen` mock component? This may make it easier to maintain with fewer components.

Reply:
I've decided to go for a simpler mock component based off @edleeks87 's comments instead and allowed for it to be open/closed 👍